src/logic/actions/ViewPortActions.ts

```diff
@@ -0,0 +1,159 @@
+import {EditorModel} from "../../staticModels/EditorModel";
+import {NumberUtil} from "../../utils/NumberUtil";
+import {ViewPointSettings} from "../../settings/ViewPointSettings";
+import {ISize} from "../../interfaces/ISize";
+import {IRect} from "../../interfaces/IRect";
+import {ImageUtil} from "../../utils/ImageUtil";
+import {RectUtil} from "../../utils/RectUtil";
+import {IPoint} from "../../interfaces/IPoint";
+import {PointUtil} from "../../utils/PointUtil";
+import {SizeUtil} from "../../utils/SizeUtil";
+import {EditorActions} from "./EditorActions";
+import {Direction} from "../../data/enums/Direction";
+import {DirectionUtil} from "../../utils/DirectionUtil";
+
+export class ViewPortActions {
+    public static updateViewPortSize() {
```

Comment:
w zasadzie jakbyś dobrze ostylował css'ami cały ten viewport, to nie musiałbyś mu ustawiać szerokości i wysokości JS-em (tak mi się wydaje)

Reply:
Ale ja tu nic nie ustawiam JSem, ja to mam ostylowane CSSem. To co tu się dzieje to zczytuję tą wartość z HTMLowego obiektu, zeby ją meić w modelu. Bo ta wartość jest mi potrzebna do róznych obliczeń.

Reply:
a, ok. w takim razie oki